orchestra/communication/views.py

```diff
@@ -35,5 +37,19 @@ def reject_staffing_request_inquiry(request,
                       {})
     if response is None:
         raise Http404
-    return render(request, 'communication/staffing_request_rejected.html',
-                  {})
+
+    next_path = request.GET.get('next')
```

Comment:
allows us to redirect back to the available task screen rather than get to a "thanks for ignoring this task" screen